src/server/controllers/usersController.js

```diff
@@ -263,6 +263,29 @@ const addJobAppliedToUser = async(req, res) => {
     res.send(201);
 };
 
+
+//Action to add user's post to their account.
+const addTimelinePost = async(req, res) => {
+    const { _id, firstname, lastname, description, timestamp} = req.body;
+    const user = await User.findByIdAndUpdate( _id , {
+        $addToSet: {
+            postsMade: { _id: _id, firstname:  firstname , lastname: lastname, description: description, timestamp: timestamp },
+        },
+    });
+    if (user) {
+        console.log("Succesfully updated awaiting connections");
+        res.status(200).json({ message: "Succesfully added user `${_id}`" });
```

Comment:
For this message log, I would maybe change it to "post from (id) saved successfully".